### ISSUE TITLE
high dpi is much requested!

### ISSUE THREAD

**Issue by rdp** [rdp/screen-capture-recorder-to-video-windows-free#50]

**Comment by bui** [comment]
bump

**Comment by DuBistKomisch** [comment]
A workaround we've found is to set the compatibility mode flag "Disable display scaling on high DPI settings" on whatever executable is loading the DLL, which in our case is `ffmpeg.exe`. This can be done programmatically via the registry, which is a bit of a pain when distributing.

**Comment by bui** [comment]
@DuBistKomisch that works, but the cursor appears to be more scaled up/pixelated. did you find a workaround for that?

**Comment by DuBistKomisch** [comment]
Nope, haven't looked into it too deeply yet. Just getting the full screen is good enough for our purposes in the short term. I'll report anything else I end up finding.
